rero_ils/modules/loans/serializers.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Loans serialization."""
+
+from invenio_records_rest.serializers.response import search_responsify
+
+from rero_ils.modules.loans.api import Loan, LoanState
+from rero_ils.modules.locations.api import Location
+from rero_ils.modules.patrons.api import Patron
+from rero_ils.modules.serializers import JSONSerializer, RecordSchemaJSONV1
+
+from ..documents.api import DocumentsSearch
+from ..items.api import Item
+from ..libraries.api import Library
+
+
+class LoanJSONSerializer(JSONSerializer):
+    """Mixin serializing records as JSON."""
+
+    def post_process_serialize_search(self, results, pid_fetcher):
```

Comment:
Load a `Loan`, a `Library` and use a `replace_refs()` for each hit of the search result not best or performance but can we do better at this time ? I don't think